src/bindings/python/src/pyopenvino/core/core.cpp

```diff
@@ -244,7 +244,21 @@ void regclass_Core(py::module m) {
         },
         py::arg("model"),
         py::arg("context"),
-        py::arg("properties"));
+        py::arg("properties"),
+        R"(
+            Creates a compiled model from a source model within a specified remote context.
+
+            GIL is released while running this function.
+
+            :param model: Model acquired from read_model function.
+            :type model: openvino.runtime.Model
```

Comment:
```suggestion
            :type model: openvino.Model
```
?
We already use this namespace in docs